app/build.gradle

```diff
@@ -77,4 +89,4 @@ task installGitHook(type: Copy) {
 
 tasks.getByPath(':app:preBuild').dependsOn installGitHook
 
-apply plugin: 'com.google.gms.google-services'
+//apply plugin: 'com.google.gms.google-services'
```

Comment:
@TacoTheDank I noticed you commented this out, same for the project `build.gradle` file. Is there a reason why this is done? 
**P.S:** The app uses Firebase Analytics

Reply:
These are in the gradle files, but nowhere in the code are they actually used. There are no instances of any firebase classes in the code, so I commented them out for now.